app/pb_kits/playbook/pb_home_address_street/docs/_home_address_street_default.html.erb

```diff
@@ -1,9 +1,10 @@
 <%= pb_rails("home_address_street", props: {
   address: "70 Prospect Ave",
   address2: nil,
```

Comment:
Are you required to pass `address2`? It should be optional

Reply:
So it is optional. I hardcoded nil, but leaving it blank would result in the same outcome -- no visible change to the kit